api/specs/api-v3.0.yaml

```
@@ -0,0 +1,1453 @@
+              description: Numéro de carte de transport
+              example: 00-MFR-6782929
+        phone:
+          type: string
+          description: Numéro au format ITU E.164
+          example: '+33601020304, +590690101010'
+        phone_trunc:
+          type: string
+          description: Numéro au format ITU E.164 tronqué des 2 derniers chiffres. Obligatoire si `phone` n'est pas renseigné.
+          example: '+336010203, +5906901010'
+        driving_license:
+            description: |
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)